conf/distro/tb-distro.conf

```diff
@@ -22,6 +22,8 @@ DISTRO_FEATURES = " \
     usbhost \
     tpm2 \
     usrmerge \
+    virtualization \
+    xen \
```

Comment:
Do we want those features in all distros or only xen? If only xen then you could do something similar to:
https://github.com/zarhus/meta-zarhus/pull/34.

Move everything except `DISTRO` to `conf/distro/include/tb-distro-common.conf`.
* `tb-distro.conf` should just have

  ```
  require conf/distro/include/tb-distro-common.conf
  
  DISTRO = "tb-distro"
  ```

* `tb-xen-distro.conf` should have:

```
  require conf/distro/include/tb-distro-common.conf
  
  DISTRO = "tb-xen-distro"
  DISTRO_FEATURES += "virtualization xen"
```

And change `distro: tb-distro` in `kas-xen-tb.yml` to `distro: tb-xen-distro`

Reply:
Resolved in c0ee7c418189d186dd29ce267160ef07bea31eb9

Very nice suggestion 👍